### PR TITLE
fix: add cosmos.msg.v1.service to message services

### PR DESCRIPTION
**Pull request by faddat**
## What is the purpose of the change

This pull request resolves an issue I encountered with using Osmosis modules in a third party chain:

        service osmosis.poolmanager.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation
        service osmosis.tokenfactory.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation
        service osmosis.poolmanager.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation
        service osmosis.tokenfactory.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation

I think that the difference between osmosis and the other chain is the use of appv2, so I checked IBC and cosmwasm, and found the annotation there.  

## Testing and Verifying

This change is already covered by existing tests.  If tests don't pass, or the change doesn't make sense to maintainers, please close this PR :)

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? no
  - [x] Changelog entry added to `Unreleased` section of `CHANGELOG.md`? shortly, yes

Where is the change documented?
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: proto/osmosis/cosmwasmpool/v1beta1/tx.proto [hunk 1]
```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 package osmosis.cosmwasmpool.v1beta1;
 
+import "cosmos/msg/v1/msg.proto";
```
[the remark this answers is not in the document]
Comment (faddat) [reply proto/osmosis/cosmwasmpool/v1beta1/tx.proto:4]: I think things are okay here Mr. Rabbit.  Will wait for the team's review though.

**Comment by faddat** [comment]
I trust commit [f607ea9](https://github.com/osmosis-labs/osmosis/pull/8689/commits/f607ea9a960b9a0f0c7c3e147be681601f94ae6e)

but not the one after it

**Comment by faddat** [comment]
```log
package github.com/osmosis-labs/osmosis/v26/app
	imports github.com/osmosis-labs/osmosis/v26/app/keepers
	imports github.com/osmosis-labs/osmosis/v26/wasmbinding
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/types
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/pool-models/balancer
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/pool-models/internal/cfmm_common
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/types: import cycle not allowed
fatal: No names found, cannot describe anything.
package github.com/osmosis-labs/osmosis/v26/app
	imports github.com/osmosis-labs/osmosis/v26/app/keepers
	imports github.com/osmosis-labs/osmosis/v26/wasmbinding
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/types
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/pool-models/balancer
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/pool-models/internal/cfmm_common
	imports github.com/osmosis-labs/osmosis/v26/x/gamm/types: import cycle not allowed
no Go files in /home/runner/work/osmosis/osmosis
make: *** [scripts/makefiles/tests.mk:50: test-sim-app] Error 1
```

This is being caused by `generated protofile changes` commits

**Comment by PaddyMc** [comment]
gm gm gm, would you mind merging main into your branch and I can start reviewing! 

We reverted => https://github.com/osmosis-labs/osmosis/pull/8710 => so the build should work now :pray:

**Comment by faddat** [comment]
oh sure thing!

**Pull request closed without merging**